examples/src/main/java/com/adobe/platform/ecosystem/examples/data/write/writer/formatter/ParquetDataFormatter.java

```diff
@@ -65,27 +65,35 @@
 
     private DataWiringParam param;
 
-    private final ParquetFieldConverter<JSONObject> fieldConverter;
-
     private final Extractor<JSONObject> extractor;
 
     private final ValidationRegistry validationRegistry;
 
     private final boolean isRegistryEnabled;
 
+    private final ParquetFieldConverter<JSONObject> jsonFieldConverter;
+
+    private final ParquetFieldConverter<List<SchemaField>> schemaFieldConverter;
+
+    private boolean isFullSchemaRequired;
+
     private static Logger logger = Logger.getLogger(ParquetDataFormatter.class.getName());
 
     public ParquetDataFormatter(ParquetIOWriter writer,
                                 DataWiringParam param,
-                                ParquetFieldConverter<JSONObject> fieldConverter,
+                                ParquetFieldConverter<JSONObject> jsonFieldConverter,
                                 Extractor<JSONObject> extractor,
-                                ValidationRegistry validationRegistry) {
+                                ValidationRegistry validationRegistry,
+                                ParquetFieldConverter<List<SchemaField>> schemaFieldConverter,
```

Comment:
Can we group constructor parameters for same interface together for better readability?